app/services/throttle.rb

```diff
@@ -53,6 +53,10 @@ class Throttle
       max_attempts: IdentityConfig.store.inherited_proofing_max_attempts,
       attempt_window: IdentityConfig.store.inherited_proofing_max_attempt_window_in_minutes,
     },
+    phone_otp: {
+      max_attempts: IdentityConfig.store.otp_delivery_blocklist_maxretry + 1,
```

Comment:
There is a very slight difference in calculating the number of max attempts where `Throttle` uses [greater than or equal to](https://github.com/18F/identity-idp/blob/2c15638b342153c3bdc60b80a70acff6013981d9/app/services/throttle.rb#L126) and `OtpRateLimiter` uses [greater than](https://github.com/18F/identity-idp/blob/2c15638b342153c3bdc60b80a70acff6013981d9/app/services/otp_rate_limiter.rb#L20)

I think we'll want to rename the config and update it to not use the `+ 1` once we've switched over.